docs/criteria/accelerators/alpha.md

```diff
@@ -9,9 +9,9 @@ All criteria must be met for the accelerator to be considered Alpha, with except
 | Accelerator | Alpha Quality | DRI Sign-off |
 | - | - | - |
 | Arrow      | ❌ |  |
-| DuckDB     | ❌ |  |
+| DuckDB     | ✅ | @peasee |
 | SQLite     | ❌ |  |
-| PostgreSQL | ❌ |  |
```

Comment:
Retroactively updating this after confirming with @sgrebnov to avoid needing a separate PR to update it.